Specs/Scene/ImplicitTilesetSpec.js

```diff
@@ -193,11 +186,222 @@ describe("Scene/ImplicitTileset", function () {
+      withExtensions.extensions["3DTILES_extension"] = {};
+
+      let metadataSchema; // intentionally left undefined
+      const implicitTileset = new ImplicitTileset(
+        baseResource,
+        withExtensions,
+        metadataSchema
+      );
+      const expected = clone(withExtensions, deep);
+      delete expected.content;
+      delete expected.extensions["3DTILES_implicit_tiling"];
+      expect(implicitTileset.tileHeader).toEqual(expected);
+    });
+
+    it("stores a template of the tile content structure", function () {
+      let metadataSchema; // intentionally left undefined
+      const implicitTileset = new ImplicitTileset(
+        baseResource,
+        implicitTileLegacyJson,
+        metadataSchema
+      );
+      expect(implicitTileset.contentHeaders[0]).toEqual(
+        implicitTileJson.content
+      );
+    });
+
+    it("allows undefined content URI", function () {
+      const noContentJson = clone(implicitTileLegacyJson);
+      delete noContentJson.content;
+      let metadataSchema; // intentionally left undefined
+      const implicitTileset = new ImplicitTileset(
+        baseResource,
+        noContentJson,
+        metadataSchema
+      );
+      expect(implicitTileset.contentUriTemplates).toEqual([]);
+    });
+
+    it("accepts tilesets with 3DTILES_bounding_volume_S2", function () {
+      const tileJson = clone(implicitTileLegacyJson, true);
+      tileJson.boundingVolume = {
+        extensions: {
+          "3DTILES_bounding_volume_S2": {
+            token: "1",
+            minimumHeight: 0,
+            maximumHeight: 100,
+          },
+        },
+      };
+      const tileJsonS2 =
+        tileJson.boundingVolume.extensions["3DTILES_bounding_volume_S2"];
+
+      let metadataSchema;
+      const implicitTileset = new ImplicitTileset(
+        baseResource,
+        tileJson,
+        metadataSchema
+      );
+      const implicitTilesetS2 =
+        implicitTileset.boundingVolume.extensions["3DTILES_bounding_volume_S2"];
+      expect(implicitTilesetS2.token).toEqual(tileJsonS2.token);
+      expect(implicitTilesetS2.minimumHeight).toEqual(tileJsonS2.minimumHeight);
+      expect(implicitTilesetS2.maximumHeight).toEqual(tileJsonS2.maximumHeight);
+    });
+
+    it("rejects bounding spheres", function () {
+      const sphereJson = {
+        boundingVolume: {
+          sphere: [0, 0, 0, 100],
+        },
+      };
+      const tileJson = combine(sphereJson, implicitTileLegacyJson);
+      let metadataSchema; // intentionally left undefined
+      expect(function () {
+        return new ImplicitTileset(baseResource, tileJson, metadataSchema);
+      }).toThrowRuntimeError();
+    });
+  });
+
+  const b3dmPattern = "https://example.com/{level}/{x}/{y}.b3dm";
+  const pntsPattern = "https://example.com/{level}/{x}/{y}.pnts";
+  const gltfPattern = "https://example.com/{level}/{x}/{y}.gltf";
+
+  describe("multiple contents", function () {
```

Comment:
same comment as above; I'm planning to properly organize this in the multiple contents PR